templates/docker-compose.yml.j2

```diff
@@ -24,4 +24,10 @@ services:
       - NET_RAW
       - NET_ADMIN
     {%- endif %}
+
+    {% if beat == 'journalbeat' %}
+    volumes:
+      - /etc/machine-id:/etc/machine-id
+      - /etc/hostname:/etc/hostname:ro
```

Comment:
Mount `/var/log/journal` also by default?

Reply:
Initially, I wanted to make the user configure where to find journals. However, we need to provide sane and secure defaults. According to the documentation journals are placed under `/var/log/journal` and if that directory does not exists, it falls back to `/run/log/journal`. What do you think about mounting both? Would that be too much?

Reply:
+1 to mount both.

Reply:
The Docker Compose file is only used to run the tests, so please do whatever you feel is best and allows the tests to pass. Conversely, please don't add anything that is _not_ needed to get the tests to pass.

Are they any Journalbeat-specific test cases that should be added?

Reply:
Oh, if this is only needed for testing the images then maybe no volume has to be mounted :slightly_smiling_face: